src/webrtc/callFeed.ts

```diff
@@ -76,6 +85,12 @@ export class CallFeed extends TypedEventEmitter<CallFeedEvent, EventHandlerMap>
     private speakingThreshold = SPEAKING_THRESHOLD;
     private speaking = false;
     private volumeLooperTimeout: ReturnType<typeof setTimeout>;
+    /**
+     * Cooldown for voice activity detection, so that we don't mute immediately when the user stops speaking
+     * But when he has been silent 100ms
+     */
+    private VADCooldownStarted = new Date();
```

Comment:
Yea I think this is more readable and easier to debug if something goes wrong. Also setTimeout could be called multiple times and I think it would become a little more complex. 

I think we should leave this as is, and potentially change it later if someone really wants to